src/powershell-native/nativemsh/pwrshplugin/pwrshclrhost.cpp

```diff
@@ -39,6 +39,13 @@ unsigned int PowerShellCoreClrWorker::LaunchClr(
 {
     // Allocate a console so that the codepage is setup correctly
     AllocConsole();
+    HWND console = GetConsoleWindow();
+    if (console != NULL)
```

Comment:
AllocateConsole returns HWND. It seems we have no need to call GetConsoleWindows.
https://learn.microsoft.com/en-us/windows/console/allocconsole